app/models/concerns/user_encrypted_attribute_overrides.rb

```diff
@@ -62,17 +64,25 @@ def stale_encrypted_email?
     encrypted_attributes[:email].stale?
   end
 
-  def phone
-    get_encrypted_attribute(name: :phone, default: nil)
+  ENCRYPTED_ATTRIBUTES.each do |attribute|
```

Comment:
For the greppability, I would suggest using `source_location`. With this neat trick, you can easily find where a method is defined by running `source_for(User, :stale_encrypted_phone?)` in the Rails console. Add the following to your `.irbrc`:
```ruby
def source_for(object, method_sym)
  if object.respond_to?(method_sym, true)
    method = object.method(method_sym)
  elsif object.is_a?(Module)
    method = object.instance_method(method_sym)
  end
  location = method.source_location
  `subl #{location[0]}:#{location[1]}` if location
  location
rescue
  nil
end
```
via https://pragmaticstudio.com/blog/2013/2/13/view-source-ruby-methods

Also, this is not the first instance of metaprogramming in this app. Devise uses this kind of metaprogramming in many places. If this module was extracted into a gem, would you still object to the use of metaprogramming?

The rule I typically use to decide when it's time to use metaprogramming is if there is more than one instance of the repetition. When we were only encrypting phone, it was fine. Now that we've introduced a second attribute, I consider that the time for reducing the duplication.